examples/gallery/lines/envelope.py

```diff
@@ -0,0 +1,121 @@
+"""
+Envelope
+--------
+The ``close`` parameter of the :meth:`pygmt.Figure.plot` method can be
+used to build a symmetrical or an asymmetrical envelope. The user can
+give either the deviations or the bounds in y direction. For the first
+case append ``"+d"`` or ``"+D"`` and for the later case ``"+b"``.
+"""
+
+
+import pandas as pd
+import pygmt
+
+# Define pandas DataFrame with columns for x and y as well as the
+# lower and upper deviations
+df = pd.DataFrame(
+    data={
+        "x": [1, 3, 5, 7, 9],
+        "y": [0.5, -0.7, 0.8, -0.3, 0.1],
+        "y_deviation_low": [0.2, 0.2, 0.3, 0.4, 0.2],
+        "y_deviation_upp": [0.1, 0.3, 0.2, 0.4, 0.1],
+    }
+)
+
+# Define the same pandas DataFrame with columns for x and y, but
+# with the lower and upper bounds
+df_bound = pd.DataFrame(
+    data={
+        "x": [1, 3, 5, 7, 9],
+        "y": [0.5, -0.7, 0.8, -0.3, 0.1],
+        "y_bound_low": [0.3, -0.9, 0.5, -0.7, -0.1],
+        "y_bound_upp": [0.6, -0.4, 1.1, 0.1, 0.2],
+    }
+)
+
+
+# Create Figure instance
+fig = pygmt.Figure()
+
+# -----------------------------------------------------------------------------
+# Left
+fig.basemap(
+    region=[0, 10, -1.5, 1.5],
+    projection="X10c",
+    frame=["WSne+tsymmetric deviation +d", "xa2f1", "ya1f0.1"],
+)
+
+# Plot a symmetrical envelope based on the deviations ("+d")
+fig.plot(
+    data=df,
+    close="+d",
+    # Fill the envelope in gray color with a transparency of 50 %
+    fill="gray@50",
+    pen="1p,gray30",
+)
+
+# Plot the data points on top
+fig.plot(
+    data=df,
+    style="c0.2c",  # Use circles with a diameter of 0.2 centimeters
+    pen="1p,gray30",
+    fill="darkgray",
+)
+
+# Shift plot origin 11 centimeters in x direction
+fig.shift_origin(xshift="11c")
+
+# -----------------------------------------------------------------------------
+# Middle
+fig.basemap(
+    region=[0, 10, -1.5, 1.5],
+    projection="X10c",
+    frame=["WSne+tasymmetric deviation +D", "xa2f1", "yf0.1"],
+)
+
+# Plot an asymmetrical envelope based on the deviations ("+D")
+fig.plot(
+    data=df,
+    fill="gray@50",
+    # Add an outline around the envelope
+    # Here, a dashed pen (+p) with 0.5-points thickness and
+    # "gray30" color is used
+    close="+D+p0.5p,gray30,dashed",
+    pen="1p,gray30",
+)
+
+# Plot the data points on top
+fig.plot(
+    data=df,
+    style="c0.2c",
+    pen="1p,gray30",
+    fill="darkgray",
+)
+
+# Shift plot origin 11 centimeters in x direction
+fig.shift_origin(xshift="11c")
+
+# -----------------------------------------------------------------------------
+# Right
+fig.basemap(
+    region=[0, 10, -1.5, 1.5],
+    projection="X10c",
+    frame=["wSnE+tbounds + b", "xa2f1", "ya1f0.1"],
```

Comment:
So far I found no way to include `+b` in the title (without a white space between `+` and `b`). Using `\\` seems not to work for this letter:
```python
import pygmt

size = 5

fig = pygmt.Figure()

for i_frame in [
        "+tmy title +b",
        "+tmy title + b",
        "+tmy title +\\z",
        "+tmy title +\\b",
]:

    fig.basemap(
        region=[-size, size, -size, size],
        projection="X" + str(size) + "c",
        frame="wsne" + i_frame,
    )
    
    fig.text(x=0, y=0, text=i_frame, font="10p")
    
    fig.plot(x=42, y=42, style="c0.1c", label=i_frame)
    fig.legend()
    
    fig.shift_origin(xshift="5.5c")

fig.show()
# fig.savefig(fname="frame_title_space.png")
```
_Output figure_:
![frame_title_space](https://github.com/GenericMappingTools/pygmt/assets/94163266/08afe736-70cd-44f3-ba6d-28f86631eee6)

Maybe it is enough to show using the deviation (i.e. ``+d``  or ``+D``), as using the bounds (``+b``) work analogous?

Reply:
It's because `+b` is a valid modifier of the `-B` option, so GMT takes `+b` as an option, rather than a normal string. `+tmy title\\053b` works.

Reply:
Thanks!